src/strategies/arbitrum/uniswapv3/strategy-univ3-eth-usdc-lp.sol

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "../../strategy-univ3-rebalance.sol";
+
+contract StrategyUsdcEthUniV3Arbi is StrategyRebalanceUniV3 {
+    address public usdc_eth_pool = 0xC31E54c7a869B9FcBEcc14363CF510d1c41fa443;
+    address[] public rewardTokens = [address(token0), address(token1)];
+
+    constructor(
+        int24 _tickRangeMultiplier,
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyRebalanceUniV3(usdc_eth_pool, _tickRangeMultiplier, _governance, _strategist, _controller, _timelock)
+    {
+        univ3_staker = 0x1f98407aaB862CdDeF78Ed252D6f557aA5b0f00d;
+
+        key = IUniswapV3Staker.IncentiveKey({
+            rewardToken: IERC20Minimal(rewardToken),
+            pool: IUniswapV3Pool(usdc_eth_pool),
+            startTime: 1633694400,
+            endTime: 1638878400,
+            refundee: 0xDAEada3d210D2f45874724BeEa03C7d4BBD41674 // rbn multisig
+        });
+
+        rewardToken = 0x6123B0049F904d730dB3C36a31167D9d4121fA6B;
+    }
+
+    function getName() external pure override returns (string memory) {
+        return "StrategyUsdcEthUniV3Arbi";
+    }
+
+    //This function assumes staking rewards is one of the deposited tokens.
+    function harvest() public override onlyBenevolent {
```

Comment:
It appears that all of this `harvest` code can be abstracted into the parent class, since none of it is specific to ETH-USDC

Reply:
I agree, but I put it in here along with a few other functions due to it will only work if the staking reward token is one of the two base tokens (so someone will review it prior to deploying). With the staking information being removed, this will go into the base class.